tests/providers/microsoft/mssql/hooks/test_mssql.py

```diff
@@ -70,3 +70,27 @@ def test_get_autocommit_should_return_autocommit_state(self, get_connection, mss
 
         mssql_get_conn.assert_called_once()
         assert hook.get_autocommit(conn) == 'autocommit_state'
+
+    @mock.patch('airflow.providers.microsoft.mssql.hooks.mssql.MsSqlHook.get_conn')
+    def test_get_uri_driver_rewrite(self, mock_get_conn):
+        mock_get_conn.return_value = PYMSSQL_CONN
+
+        hook = MsSqlHook()
+        res_uri = hook.get_uri()
+
+        mock_get_conn.assert_called_once()
+        assert res_uri == (
+            "mssql+pymssql://"
+            f"{PYMSSQL_CONN.login}:{PYMSSQL_CONN.password}"
+            f"@{PYMSSQL_CONN.host}:{PYMSSQL_CONN.port}/{PYMSSQL_CONN.schema}"
+        )
+
+    @unittest.skipIf(PY38, "Mssql package not available when Python >= 3.8.")
```

Comment:
Is this right?
https://github.com/pymssql/pymssql/pull/659

Reply:
You are right, version 2.1.5 added support for Python 3.8 !

https://github.com/pymssql/pymssql/releases/tag/v2.1.5

This is the version listed in airflow constraints file https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-3.8.txt

We should remove the decorator for the other tests as well.

Reply:
https://github.com/apache/airflow/pull/25214

Reply:
I rebased and removed mine as well @eladkal